### PR TITLE
norm 1.5r5

### PR DESCRIPTION
**Pull request by duedal**
Updated norm to the new release @bebopagogo mentioned in #41051

Thanks!

### CODE REVIEW

File: Library/Formula/norm.rb [hunk 1]
```diff
@@ -1,8 +1,9 @@
 class Norm < Formula
   desc "NACK-Oriented Reliable Multicast"
   homepage "https://www.nrl.navy.mil/itd/ncs/products/norm"
-  url "https://downloads.pf.itd.nrl.navy.mil/norm/archive/src-norm-1.5b3.tgz"
-  sha256 "ec2014f57be12ea3ade0685faa1e6e161f168311c8975093bce33a6e1dc5e77c"
+  url "https://downloads.pf.itd.nrl.navy.mil/norm/archive/src-norm-1.5r5.tgz"
+  version "1.5r5"
```
Comment (DomT4) [inline Library/Formula/norm.rb]: What `version` is it reading here without the manual `version` line?

Comment (duedal) [inline Library/Formula/norm.rb]: Unfortunately it detects "5" as the version.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/4d674e2239862695eae5cdcf2ca2b2a548bdfd30. Thank you for your contribution to Homebrew, we appreciate it! :smiley_cat:

File: Library/Formula/norm.rb (hunk 1 above)
Comment (bebopagogo) [inline Library/Formula/norm.rb]: FYI - I have now posted src-norm-1.5r6.tgz that has an important bug fix.

For future reference, is there a certain way I should report such key updates to homebrew?

best regards,

Brian

Comment (duedal) [inline Library/Formula/norm.rb]: @bebopagogo I've created a pull request #41554 - I can't speak for the homebrew team, but I guess the best way to do it is by submitting a pull request like I did.
I'm happy to do it, but you can of course also do it yourself; https://git.io/brew-docs

Comment (DomT4) [inline Library/Formula/norm.rb]: Yeah, essentially PRs are forever welcome. Easiest way to ensure something gets a timely update.

**Pull request closed without merging**